src/bin/am/commands/start.rs

```diff
@@ -254,10 +255,25 @@ pub async fn handle_command(args: CliArguments, config: AmConfig, mp: MultiProgr
         args.metrics_endpoints.push(endpoint);
     }
 
+    let (prometheus_tx, prometheus_rx) = oneshot::channel();
```

Comment:
Would it make sense to use something like `tokio::sync::watch` instead of `oneshot` channels? (see: https://docs.rs/tokio/latest/tokio/sync/watch/index.html)

That way we do not have to add more `oneshot` transmitters when we need another consumer for the listen address.